perl/p5-file-flock-tiny/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.26 5.28 5.30 5.32 5.34
+perl5.setup         File-Flock-Tiny 0.14
+#perl5.setup         File-Flock-Tiny 0.14 ../by-authors/id/Z/ZW/ZWON
```

Comment:
please pick the correct one and remove uncommented lines

Reply:
I don't know the significance of adding `../by-authors/id/Z/ZW/ZWON`, but since you didn't ask me to use it I'll assume that it's not needed. Please let me know if that's not the case.

Reply:
resolved with commit [64f38ea](https://github.com/macports/macports-ports/pull/16319/commits/64f38eaa183205ed9250bd023e8463215c200ad8)